keyboards/lendunistus/rpneko65/rev1/readme.md

```diff
@@ -0,0 +1,24 @@
+# rpneko65
+
+![rpneko65](https://i.imgur.com/QUIUSqJ.png)
```

Comment:
```suggestion
![rpneko65](https://i.imgur.com/QUIUSqJh.png)

```

This renders a lower resolution version that is a bit more friendly to github markdown.